src/lib/common/sol-mainloop.c

```diff
@@ -76,6 +76,10 @@ extern void sol_comms_shutdown(void);
 extern int sol_update_init(void);
 extern void sol_update_shutdown(void);
 #endif
+#ifdef USE_UPDATE
+extern int sol_ipm_init(void);
+extern void sol_ipm_shutdown(void);
+#endif
```

Comment:
On 09-05-2016 16:01, Gustavo Sverzut Barbieri wrote:
> In src/lib/common/sol-mainloop.c
> <https://github.com/solettaproject/soletta/pull/1996#discussion_r62553975>:
>
>> @@ -76,6 +76,10 @@ extern void sol_comms_shutdown(void);
>>  extern int sol_update_init(void);
>>  extern void sol_update_shutdown(void);
>>  #endif
>> +#ifdef USE_UPDATE
>> +extern int sol_ipm_init(void);
>> +extern void sol_ipm_shutdown(void);
>> +#endif
>
> what about my proposal to:
>
> #else
> static inline int
> sol_ipm_init(void)
> {
>     return 0;
> }
>
> static inline void
> sol_ipm_shutdown(void)
> {
> }
> #endif

Omg... I knew I was forgetting something =/

>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/solettaproject/soletta/pull/1996/files/64ac7ea3f07d3a26a3d264869f28dc376ba64cc9#r62553975>
>